### PR TITLE
Update: Add not between operator to elide

### LINKED ISSUE #1044: Support `nbet` operator

### PR DESCRIPTION
**Pull request by holderb5**
Resolves #1044 

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/data/addon/adapters/facts/elide.ts [hunk 1]
```diff
@@ -78,6 +78,10 @@ export default class ElideFactsAdapter extends EmberObject implements NaviFactAd
         return `${fieldStr}=ge=(${filterVals[0]});${fieldStr}=le=(${filterVals[1]})`;
       }
 
+      if (operator === 'nbet') {
+        return `${fieldStr}=le=(${filterVals[0]});${fieldStr}=ge=(${filterVals[1]})`;
```
Comment (jkusa) [inline packages/data/addon/adapters/facts/elide.ts:82]: Should this be:
```suggestion
        return `${fieldStr}=lt=(${filterVals[0]}),${fieldStr}=gt=(${filterVals[1]})`;
```

Comment (mattgraham1995) [inline packages/data/addon/adapters/facts/elide.ts:82]: I think this should be `lt` and `gt`. I believe no rows that match values within the supplied interval should be included

Comment (mattgraham1995) [reply packages/data/addon/adapters/facts/elide.ts:82]: Oops, yeah what Jon said

Comment (jkusa) [reply packages/data/addon/adapters/facts/elide.ts]: Don't we want an `OR`?

Comment (holderb5) [reply packages/data/addon/adapters/facts/elide.ts]: Yup you're correct, would that just be || in this case?

Comment (jkusa) [reply packages/data/addon/adapters/facts/elide.ts]: The suggestion above is the correct syntax ^^

**Review by jkusa — APPROVED** [review]

**Review by mattgraham1995 — APPROVED** [review]

**Pull request merged**